docs/en/reference/data-retrieval-and-manipulation.rst

```diff
@@ -530,35 +530,3 @@ given data.
     <?php
     $conn->update('user', array('username' => 'jwage'), array('id' => 1));
     // UPDATE user (username) VALUES (?) WHERE id = ? (jwage, 1)
-
-By default the Doctrine DBAL does no escaping. Escaping is a very
```

Comment:
I don't really understand the purpose of this block. The above `UPDATE` statement doesn't use `quote()` or `quoteIdentifier()`.

Reply:
They simply don't relate to each other I think. This block comes from e3ddd6 and was introduced in another document, and they were stuck together in d7a52291f .

When it was introduced in e3ddd6, it was as an introduction to both `quote` and `quoteIdentifier`